pycroft/helpers/printing/__init__.py

```diff
@@ -289,7 +290,7 @@ def generate_user_sheet(
             six monthly contributions at once.'''.format(
             contribution / 100), style['JustifyText']))
 
-        recipient = 'Studierendenrat TUD - AG DSN'
+        recipient = bank_account.owner
```

Comment:
Perhaps it would be wise to emit a warning in case `owner` is `None ` so we see it in sentry. Sure, it shouldn't happen in practice, but the types don't quite match, right? So in particular perhaps set recipient to the empty string in that case so we can be sure the rendering doesn't break.

Reply:
I've set the server_default in the migration to `""` now.  This won't work for exporting non-attributable transfers though, some longer name has to be set.